stacker/actions/build.py

```diff
@@ -96,17 +101,19 @@ def resolve_parameters(parameters, blueprint, context, provider):
                         provider.get_output(stack_fqn, output))
                 except KeyError:
                     raise exceptions.OutputDoesNotExist(stack_fqn, v)
-            value = ",".join(v_list)
-        if value is None:
-            logger.debug("Got None value for parameter %s, not submitting it "
-                         "to cloudformation, default value should be used.",
-                         k)
-            continue
-        if isinstance(value, bool):
-            logger.debug("Converting parameter %s boolean \"%s\" to string.",
-                         k, value)
-            value = str(value).lower()
-        params[k] = value
+            elif v is None:
+                logger.debug("Got None value for parameter %s, not submitting "
+                             "it to cloudformation, default value should be "
+                             "used.", k)
+                continue
+            elif isinstance(v, bool):
+                logger.debug("Converting parameter %s boolean \"%s\" to "
+                             "string.", k, v)
+                v_list.append(str(v).lower())
+            else:
+                v_list.append(v)
+        if v_list != []:
```

Comment:
@phobologic I think the `if` does match the old behavior, specifically the way `None`s have [been handled here](https://github.com/remind101/stacker/blob/02a5ad5c202be1ec8a4247a2984af599db9e17ef/stacker/actions/build.py#L104).

If a value is always returned, then it trips [test_resolve_parameters_none_conversion](https://github.com/remind101/stacker/blob/02a5ad5c202be1ec8a4247a2984af599db9e17ef/stacker/tests/actions/test_build.py#L267-L278).